packages/components/table/stories/table.stories.js

```diff
@@ -0,0 +1,102 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { withKnobs, array } from '@storybook/addon-knobs';
+
+import '@tradeshift/elements.table';
+
+storiesOf('ts-table', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const cols = array('cols', [
+			{
+				visibility: 'desktop',
+				type: 'document',
+				property: 'context',
+				size: 'large',
+				value: 'Context'
+			},
+			{ visibility: 'desktop', type: 'status', property: 'type', size: 'small', value: 'Type', display: 'left' },
+			{ visibility: 'desktop', property: 'participants', size: 'small', value: 'Participants' },
+			{ visibility: 'desktop', property: 'newActivity', value: 'New Activity' },
+			{
+				visibility: 'desktop',
+				property: 'lastActivity',
+				value: 'Last Activity',
+				size: 'medium',
+				display: 'right'
+			},
+			{ visibility: 'mobile', type: 'document', property: 'mobileContext' },
+			{
+				visibility: 'mobile',
+				property: 'mobileLastActivity',
+				renderFunction: function(item) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use shorthand for method "renderFunction". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS3498)